### ISSUE TITLE
incorrect work function $pods->field( "xxx.term_id" )

### ISSUE THREAD

**Issue by HectroPrima** [pods-framework/pods#2747]
Create a Custom Post Type the name of demo
and Custom Taxonomy named demo1
associate their demo uses demo1

in version 2.4.3-b-1 code works and provides a list of related records from demo1

```
$pods = pods('demo', array('limit' => -1));
while ( $pods->fetch() ) {
    foreach ( $pods->field( "demo1.term_id" ) as $host ) {
        echo "demo1: ".$host."<br>";
    }
}
```
Version 2.5.1 provides ID is always the first record of the demo1
all this because of the replacement file PodsAPI.php

old version
```
                foreach ( $taxonomies as $taxonomy ) {
                    $fields[ $taxonomy->name ] = array(
                        'name' => $taxonomy->name,
                        'label' => $taxonomy->labels->name,
                        'type' => 'taxonomy',
                        'pick_object' => 'taxonomy',
                        'pick_val' => $taxonomy->name,
                        'alias' => array(),
                        'hidden' => true,
                        'options' => array(
                            'taxonomy_format_type' => 'multi'
                        )
                    );
                }
```
new version
```
                foreach ( $taxonomies as $taxonomy ) {
                    $fields[ $taxonomy->name ] = array(
                        'name' => $taxonomy->name,
                        'label' => $taxonomy->labels->name,
                        'type' => 'taxonomy',
                        'pick_object' => 'taxonomy',
                        'pick_val' => $taxonomy->name,
                        'alias' => array(),
                        'hidden' => true,
                        'options' => array(
                            'pick_format_type' => 'multi'
                        )
                    );
                }
```

'taxonomy_format_type' => 'multi'
replaced by
'pick_format_type' => 'multi'

what to do?
how to get a list of related records?

**Comment by HectroPrima** [comment]
this https://github.com/pods-framework/pods/commit/1bc1eeeed6e66074307cf3f01fc2cfbcbe4a440c commit

**Comment by penoonan** [comment]
For those like me who got this far before the fix was released, remember you can fall back to using native WP query functions to get this functionality. In my case, for a taxonomy "voltage" on a post of type "product", `$product->field('voltage')` was only returning one result when it should have returned several. I was only able to change that to `wp_get_post_terms($product->id(), 'voltage')` to get all the needed results.

**Comment by Shelob9** [comment]
Fixed via #2750
